### DISCUSSION TITLE
DMs are hard to see for some people

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#19727] [Ideas] [answered]
AFAIK (I could be wrong) currently when you send someone a DM it adds 1 to the notification count on the top right but if you're currently in a multiplayer game then you probably download a lot of new beatmaps all the time so there will be a lot of "beatmap downloaded" notifications and so one doesn't really check their notifications. Anyhow, some people definitely seem to easily miss DMs sometimes. I was wondering if you could make DMs pop into the screen when you get them? Not if you're in a game of course.

Maybe notifications can have a priority level and DMs have higher priority and show up differently.

Ignore this if DMs are already visible enough and my friend is just blind.

**Comment by peppy** [comment, marked as answer]
The end goal is to have the same interface as the web site.
